autoPyTorch/datasets/resampling_strategy.py

```diff
@@ -76,8 +83,14 @@ def is_stratified(self) -> bool:
         return getattr(self, self.name) in stratified
 
 
+class NoResamplingStrategyTypes(IntEnum):
+    no_resampling = 8
+    shuffle_no_resampling = 9
+
+
```

Comment:
It is likely that we need to adapt this part to the current development branch, but when do we modify?
After mergint?